Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -242,4 +242,15 @@
   - type: Sprite
     sprite: DeltaV/Clothing/Uniforms/Jumpsuit/kilt.rsi
   - type: Clothing
-    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/kilt.rsi
\ No newline at end of file
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/kilt.rsi
+
+- type: entity  #Delta-V Chemistry jumpsuit variant
```

```suggestion
- type: entity
```
No need for a change marker since it is in a DeltaV file.